prime_factor/prime_factor.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <limits.h>
+
+int generate_primes(int num) {
+    int cnt = 0;
+    printf("Prime Factors: ");
```

Comment:
> We can put an upper bound to the number of possible primes for a given number.

And that can be used as array size for the elements. Instead of 1000 or 60000.